extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -185,6 +196,28 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
       productForPdp = shoppingDataPerPage[0];
     }
 
+    //pdp view
+    if (productForPdp) {
+      this.variableService_.onVariableUpdate(
+        AnalyticsVariable.STORY_SHOPPING_PRODUCT_ID,
+        productForPdp.productId
+      );
+    }
+    //plp view with more than one product on the page
```

Comment:
Please add a space between the slashes and the first word. Please also add a period to the end of the comment.